dali/base/dadfs.cpp

```diff
@@ -7466,8 +7470,17 @@ class CNamedGroupIterator: implements INamedGroupIterator, public CInterface
 public:
     IMPLEMENT_IINTERFACE;
     CNamedGroupIterator(IRemoteConnection *_conn,IGroup *_matchgroup=NULL,bool _exactmatch=false)
-        : conn(_conn), matchgroup(_matchgroup)
+        : conn(_conn)
     {
+        // the matchgroup may contain ports, but they are never part of published groups and are not to be used for matching
+        SocketEndpointArray epa;
+        for (unsigned i=0; i<_matchgroup->ordinality(); i++)
```

Comment:
check for _matchgroup == NULL before dereferencing ?

Reply:
it can't be null in this context. It is created (a CNameGroupItrator with IGroup as param) when there is a grp

Reply:
my mistake, there is a use case when it uses defaults and _matchGroup is null (and _exactMatch is false).
There should be a check for null. I'll change it.